doc/deprecated.rst

```diff
@@ -26,6 +26,13 @@ Extensions
 * As of Twig 1.x, the ability to remove an extension is deprecated and the
   ``Twig_Environment::removeExtension()`` method will be removed in 2.0.
 
+* As of Twig 1.x, the ``Twig_ExtensionInterface::initRuntime()`` is deprecated.
```

Comment:
Please say ``As of Twig 1.23``. ``As of 1.x`` does not make sense as ``1.x`` is not a version